src/react/features/connectors/effects/useGetRoomEffect.js

```diff
@@ -12,10 +12,10 @@ export const useGetRoomEffect = (connectorId) => {
 
   const { dispatch, dispatchPromiseAction } = useDispatchActions()
   useEffect(() => {
-    connect.forEach(({ address }) => {
+    connect.forEach(({ rootId, diograph, address }) => {
       dispatchPromiseAction(
         () => invokeChannel(channels.GET_ROOM, { address }),
-        () => getRoom({ address })
+        () => getRoom({ rootId, diograph, address })
       )
```

Comment:
We could set focus here (I hope):
.then(({ rootId }) => dispatch(setFocus({ focus: rootId })

Reply:
I don't have good enough overall view about this architecture to say anything about this :/
Maybe we could or maybe current is good enough? 🙄 

My current goal is just to 
1) make progress in feature-wise
2) improve the test coverage so if we want to change something, refactoring would be nice and easy against green tests that won't break (concretely the E2E tests)
3) Little by little get a good grasp of the current code base and be able to propose / have something to say about architectural stuff. Until then I take all the existing stuff as granted and "good" because I don't know anything better...